src/mocks/testHandlers.ts

```diff
@@ -8,3 +8,12 @@ export const setupFailedNetworkRequest = (
     return res(ctx.status(statusCode));
   });
 };
+
+export const setupFailedNetworkPostRequest = (
+  url: string,
+  statusCode: number
+): RestHandler<MockedRequest<DefaultBodyType>> => {
+  return rest.post(url, (req, res, ctx) => {
```

Comment:
Not sure if it supports this, but can we do a request and pass in the type e.g. GET or POST via a paremeter?